SemanticsTests.fs

```diff
@@ -405,13 +569,6 @@ module CommandTests =
                 (Array
                     (Array (Int (), Some 320, ()), Some 240,
                      AVar (Reg (After "grid"))),
-                (Array
```

Comment:
These have gone because the new framer doesn't make these redundant intermediates